### PR TITLE
Update data refresh to use media tables

### LINKED ISSUE #2093: Update ingestion server to copy data from the catalog media tables
## Description

Once the `image` and `audio` tables contain up-to-date standardized popularity data, we can update the data refresh to read from these tables rather than their respective materialized views. Because the schema of the media tables now matches what the ingestion server already expects, we can simply change the table name each place it is referenced:

- [`_get_shared_cols`](https://github.com/WordPress/openverse/blob/003c0d3e5c2813b671120144756659775f3369f4/ingestion_server/ingestion_server/ingest.py#L75)
- [`refresh_api_table`](https://github.com/WordPress/openverse/blob/003c0d3e5c2813b671120144756659775f3369f4/ingestion_server/ingestion_server/ingest.py#L310)
- [`get_copy_data_query`](https://github.com/WordPress/openverse/blob/003c0d3e5c2813b671120144756659775f3369f4/ingestion_server/ingestion_server/queries.py#L206)

## Additional context

This must NOT be merged until the media tables are fully backfilled with popularity data. For that to be true:
* popularity must be being calculated at ingestion
* the popularity_refresh DAGs must have both completed an initial DagRun successfully

### LINKED ISSUE #2094: Remove popularity steps from the data refresh DAGs
## Description

Popularity constants are recalculated and the matviews are dropped/recalculated during data refreshes. Once popularity is stored in the media tables, backfilled, and a data refresh has successfully run against the media tables, we can remove these steps.

As part of this step, we can also manually drop the matviews and remove all references to them from the code. This includes the matview recreation steps in the `recreate_<media_type>_popularity_calculation` DAGs.

## Additional context

Blocked on the completion of a data refresh from the media tables.

### PR DESCRIPTION
**Pull request by stacimc**
## Fixes

Fixes #2093 by @stacimc 
Fixes #2094 by @stacimc 

## Description

* Removes popularity calculation steps from the data refresh DAGs
* Updates the ingestion server to copy from the media tables, rather than the matviews

Most of the changes in the ingestion server add handling for the names of the upstream and downstream tables being different. Previously, we could always append `_view` to the end of the downstream table name to get the upstream table name. Now, we must do this for `audioset` but not for the media tables. (I.e.: the relevant tables in the API are `audio`, `image`, `audioset`. The corresponding catalog ones are `audio`, `image`, `audioset_view`).

When this is merged, data refreshes and popularity refreshes will officially be fully decoupled and we'll be able to drop the matviews.

This PR does not remove the matview locally or do other clean-up. This will come in a subsequent PR. The intention is to keep this PR as small as possible for review.

## Testing Instructions

Run `just recreate` from this branch. This will give you changes needed to the schema of the `audio` table and `audioset_view` and update your sample data. (These are prerequisite changes in `main`, which this branch has been rebased on).

Run a few provider DAGs to get some new data. I ran jamendo for a minute or two until I got 200 new audio records and several new audiosets, and SMK to get some images.

Run `just catalog/pgcli` and manually check the `COUNT` for each of the following:

```sql
SELECT COUNT FROM image;

SELECT COUNT FROM audio;

SELECT COUNT FROM audio WHERE audio_set_foreign_identifier IS NOT NULL;

SELECT COUNT FROM audioset_view;
```

Then run an image data refresh and audio data refresh locally and ensure they pass. Make sure you have the `create_filtered_image_index` and `create_filtered_audio_index` DAGs enabled before running the data refreshes.

Now confirm that the records are available in the API: run `just api/pgcli` and run the same `SELECT COUNT` queries you just ran in the catalog. The counts should all be identical.

**NOTE** This PR does not drop the materialized views. You can see that the records are being copied from the media tables by inspecting the logs in the ingestion server, but to be ABSOLUTELY certain you can run this in your catalog DB:

```psql
DROP MATERIALIZED VIEW audio_view;
DROP MATERIALIZED VIEW image_view;
```

And then re-run the data refreshes and ensure they still work.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by stacimc** [comment]
Updated priority to at least `high` as this is now one of the things blocking ingestion and refreshes in the catalog.

**Comment by sarayourfriend** [comment]
Reviewing this now @stacimc. Thank goodness I know how to properly run the Jamendo DAG now :sweat_smile:

**Review by sarayourfriend — APPROVED** [review]
This is really amazing. Not just seeing an exciting project coming to fruition with such astounding benefits, but also to see how significantly this simplifies the data refresh process. Kudos, Staci!

Tested this locally and it works perfectly. I followed your instructions and was able to get the new results in my local API without issue. I ran into some problems with my local Airflow configuration that was for some reason pointed at the wrong Elasticsearch port :facepalm: Go figure. Sorted that out and things worked great.

File: catalog/DAGs.md [hunk 1]
```diff
@@ -202,24 +202,14 @@ airflow dags trigger --conf
...
-Once this step is complete, the data refresh can be initiated. A data refresh
-occurs on the Ingestion server in the openverse project. This is a task which
-imports data from the upstream Catalog database into the API, copies contents to
-a new Elasticsearch index, and finally makes the index "live". This process is
-necessary to make new content added to the Catalog by our provider DAGs
-available to the API. You can read more in the
+given media type these DAGs will initiate a data refresh on the ingestion server
+and await the success or failure of that task.
+
+A data refresh occurs on the Ingestion server in the openverse project. This is
```
Comment (obulat) [inline catalog/DAGs.md:208]:
```suggestion
A data refresh occurs on the Ingestion server in the Openverse project. This is
```

File: catalog/DAGs.md [hunk 2]
```diff
@@ -535,24 +525,14 @@ This script can be run either to ingest the full dataset or as a dated DAG.
...
```
Comment (obulat) [inline catalog/DAGs.md:531]:
```suggestion
A data refresh occurs on the Ingestion server in the Openverse project. This is
```

File: catalog/dags/data_refresh/dag_factory.py [hunk 3]
```diff
@@ -1,20 +1,10 @@
...
-initiating a data refresh, the DAG will first refresh the view in order to
-update popularity data for records that have been ingested since the last refresh.
-On the first run of the month, the DAG will also refresh the underlying tables,
-including the percentile values and any new popularity metrics. The DAG can also
-be run with the `force_refresh_metrics` option to run this refresh after the first
-of the month.
-
-Once this step is complete, the data refresh can be initiated. A data refresh
-occurs on the Ingestion server in the openverse project. This is a task
+A data refresh occurs on the Ingestion server in the openverse project. This is a task
```
Comment (obulat) [inline catalog/dags/data_refresh/dag_factory.py:7]:
```suggestion
A data refresh occurs on the Ingestion server in the Openverse project. This is a task
```

File: ingestion_server/ingestion_server/ingest.py [hunk 4]
```diff
@@ -55,24 +55,24 @@
 #: the port of the upstream DB from the POV of the downstream DB
 
 
-def _get_shared_cols(downstream, upstream, table: str):
+def _get_shared_cols(downstream, upstream, upstream_table: str, downstream_table: str):
```
Comment (obulat) [inline ingestion_server/ingestion_server/ingest.py:58]: Is there a type we can add to `downstream` and `upstream`?

**Review by obulat — APPROVED** [review]
Tested locally after dropping the materialized views - and everything works perfectly! I've added some comments on the documentation.

**Review by krysal — APPROVED** [review]
Works perfectly 👏 Very impressive to see how much the data refresh is being simplified. Awesome work, @stacimc!

File: catalog/DAGs.md (hunk 1 above)
Comment (stacimc) [reply catalog/DAGs.md]: Oops, good catch!

**Pull request merged**